controllers/networking-calico/pkg/apis/calico/types_network.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package calico
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type Backend string
+
+const (
+	Bird Backend = "bird"
+	None Backend = "None"
```

Comment:
Shouldn't this be `none`?
https://docs.projectcalico.org/v3.8/reference/node/configuration

Or have `Bird` be `Bird` and then lower case it in the implemenetation

Reply:
true, nice catch. Its correct in the versioned API but not here.